k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -1266,6 +1274,32 @@ private void processMessageToForward(MessageViewInfo messageViewInfo) throws Mes
         attachmentPresenter.processMessageToForward(messageViewInfo);
     }
 
+    private void processMessageToForwardAsAttachment(MessageViewInfo messageViewInfo) throws MessagingException {
+        Message message = messageViewInfo.message;
+
+        String subject = message.getSubject();
+        if (subject != null && !subject.toLowerCase(Locale.US).startsWith("fwd:")) {
+            mSubjectView.setText("Fwd: " + subject);
+        } else {
+            mSubjectView.setText(subject);
+        }
+
+        // "Be Like Thunderbird" - on forwarded messages, set the message ID
+        // of the forwarded message in the references and the reply to.  TB
+        // only includes ID of the message being forwarded in the reference,
+        // even if there are multiple references.
+        if (!TextUtils.isEmpty(message.getMessageId())) {
+            mInReplyTo = message.getMessageId();
+            mReferences = mInReplyTo;
+        } else {
+            if (K9.DEBUG) {
+                Log.d(K9.LOG_TAG, "could not get Message-ID.");
+            }
```

Comment:
i didn't want to modify the code, its copied from a few lines above - see function processMessageToForward.

if we need to change it, then we should also modify it above.
what do you think?